packages/reactivated/src/shell.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+const serJSON = (data: any): string => {
+    return JSON.stringify(data).replace(/</g, "\\u003c");
+};
+
+export const PageShell = (
+    props: React.PropsWithChildren<{
+        vite: string;
+        mode: "production" | "development";
+        preloadContext: any;
+        preloadProps: any;
+        entryPoint: string;
+    }>,
+) => {
+    const scriptNonce = props.preloadContext.request.csp_nonce
+        ? `nonce="${props.preloadContext.request.csp_nonce}"`
```

Comment:
There's a problem here where <style> tags like so:

```
<style type="text/css">{`""`}</style>
```

On the server, they come as escaped and on the client they don't. So there's a mismatch and the whole hydration fails. 

I saw this thread https://github.com/vitejs/vite/issues/15662 but notice we don't use transform on our output at all. So not sure what the deal is.

Reply:
Like so:

```
            <style type="text/css">{`
                body {
                    background-color: "red";
                }
            `}</style>
```

Add just that as the sole content of a page to see the mismatch.

Reply:
Might actually be a react bug, but I don't see a lot of traction on this ticket:

https://github.com/facebook/react/issues/26946

I can't possibly be one of a handful of users that has inline styles with quotes.